lighthouse-cli/bin.ts

```diff
@@ -116,6 +120,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('disable-cpu-throttling', true)
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
+  .default('port', 9222)
```

Comment:
would you want `chrome-debug` to support port 0 (aka give me a random port)? Not that useful, since the caller will have to know the port in order to talk to it on that port. If not, this is an easy fix and I can add it